### ISSUE TITLE
Check duplication for partnership and display of parents and children

### ISSUE THREAD

**Issue by cynthia5610** [star4z/familytree#87]
Added two partnerships with same child. The child detail page would display the same parents twice. Should check for duplication.

**Comment by star4z** [comment]
I don't think that this is necessarily wrong behavior on the part of the program. I think it would be more helpful to show a check to make sure the user did that on purpose, rather than preventing the user from doing that entirely.
